<?xml version="1.0" encoding="utf-8"?><messages><msg code="0" type="2">Start Time: Monday, April 15, 2024 2:33:12 PM</msg><msg code="0" type="0">
 1. Creating outputs
</msg><msg code="0" type="0">    - Spans</msg><msg code="0" type="0">&lt;geoprocessing spatial reference object object at 0x00000175AD1DFB10&gt;</msg><msg code="0" type="0">    - Structures</msg><msg code="0" type="0">    - Sections</msg><msg code="0" type="0">getting arc pro list</msg><msg code="0" type="0">ETL.1840</msg><msg code="0" type="0">getting arc pro list</msg><msg code="0" type="0">ETL.1840</msg><msg code="-2147467259" type="100">Traceback (most recent call last):
  File "D:\Xml-to-layer-arc-tool\xml_to_layer.py", line 403, in &lt;module&gt;
    main()
  File "D:\Xml-to-layer-arc-tool\xml_to_layer.py", line 396, in main
    create_sections_feature_from_OH_conductor(dst_spans, dst_gdb, get_arc_pro_list(), sr=xml_sr )
  File "D:\Xml-to-layer-arc-tool\xml_to_layer.py", line 306, in create_sections_feature_from_OH_conductor
    for row in cursor:
RuntimeError: An expected Field was not found or could not be retrieved properly. [arc_sections]
</msg><msg code="0" type="0">Failed script Script...</msg><msg code="-2147467259" type="100">Failed to execute (Script).</msg><msg code="0" type="3">Failed at Monday, April 15, 2024 2:34:07 PM (Elapsed Time: 54.52 seconds)</msg></messages>